static/js/components/EmploymentTab.js

```diff
@@ -168,7 +168,7 @@ class EmploymentTab extends ProfileTab {
           colored
           onClick={this.addWorkHistoryEntry}
           key="I'm unique!"
-          className={`add-employment-button ${fabSpacing()}`}>
+          className={`profile-add-button ${fabSpacing()}`}>
```

Comment:
I'm absolute positioning the FABButton there, I'll update this for the employment tab too